cms/static/js/spec/views/pages/library_users_spec.js

```diff
@@ -87,27 +87,27 @@ function ($, AjaxHelpers, ViewHelpers, ManageUsersFactory, ViewUtils) {
                 expect(requests.length).toEqual(0);
             });
 
-            // TODO: This test has become flaky. See TNL-3166
-            xit("displays an error when the user has already been added", function () {
+            it("displays an error when the user has already been added", function () {
                 var requests = AjaxHelpers.requests(this);
+                var promptSpy = ViewHelpers.createPromptSpy();
                 $('.create-user-button').click();
                 $('.user-email-input').val('honor@example.com');
-                var warningPromptSelector = '.wrapper-prompt.is-shown .prompt.warning';
-                expect($(warningPromptSelector).length).toEqual(0);
+                expect($('.wrapper-prompt.is-shown .prompt.warning').length).toEqual(0);
```

Comment:
Is there value in testing this? Since it doesn't seem that we can verify that an element with this selector exists after the dialog is shown, there is no way to know if this css actually corresponds to anything in the dialog. I would suggest just removing this line.